client/state/jetpack-connect/reducer/jetpack-auth-attempts.js

```diff
@@ -6,26 +6,29 @@ import { AUTH_ATTEMPS_TTL } from '../constants';
 import { isStale } from '../utils';
 import { JETPACK_CONNECT_COMPLETE_FLOW, JETPACK_CONNECT_RETRY_AUTH } from 'state/action-types';
 import { jetpackAuthAttemptsSchema } from './schema';
+import { keyedReducer } from 'state/utils';
 
-export default function jetpackAuthAttempts( state = {}, action ) {
-	switch ( action.type ) {
+export function authAttempts( state = undefined, { type, attemptNumber } ) {
```

Comment:
It doesn't make sense to pass in the next attemptNumber, but I'd rather leave that for a follow-up PR. There's enough in here.